site/pages/cookie-details.tsx

```diff
@@ -0,0 +1,157 @@
+import Link from "next/link";
+import { ReactElement } from "react";
+import { TwoThirdsLayout } from "../components/layout/Layout";
+
+const title = "Cookie details - Create Transport Disruption Data Service";
+const description = "Cookie details page for the Create Transport Disruption Data Service";
+
+const Contact = (): ReactElement => {
+    return (
+        <TwoThirdsLayout title={title} description={description}>
+            <h1 className="govuk-heading-xl">Details about cookies on the Create Transport Disruption Data Service</h1>
+            <p className="govuk-body">
+                The Create Transport Disruption Data Service puts small files (known as ‘cookies’) onto your computer to
+                collect information about how you browse the site. Find out more about the cookies we use, what they’re
+                for and when they expire.
+            </p>
+            <h2 className="govuk-heading-m">Tracking</h2>
```

Comment:
We can remove this tracking section for the time being since I'm not sure we'll be using GA

Reply:
done